Cargo.toml

```diff
@@ -93,18 +93,21 @@ starknet-crypto = { version = "0.6.1", default-features = false }
 starknet_api = { version = "0.7.0-dev.0", default-features = false }
 
 # Ethereum dependencies
-alloy-primitives = "0.7.0"
+alloy-primitives = "0.7.2"
 alloy-rlp = { version = "0.3.4", default-features = false }
-alphanet-instructions = { git = "https://github.com/paradigmxyz/alphanet.git", rev = "87be409", default-features = false }
+alphanet-instructions = { git = "https://github.com/paradigmxyz/alphanet.git", rev = "a7f66d6", default-features = false }
 ethers = { version = "2.0.9", default-features = false }
 ethers-solc = { version = "2.0.9", default-features = false }
 jsonrpsee = { version = "0.21.0", features = ["macros", "server"] }
-reth-primitives = { git = "https://github.com/paradigmxyz/reth.git", tag = "v0.2.0-beta.6", default-features = false, features = [
-  "arbitrary",
+reth-primitives = { git = "https://github.com/paradigmxyz/reth.git", tag = "v0.2.0-beta.7", default-features = false, features = [
+  "alloy-compat"
 ] }
-reth-rpc-types = { git = "https://github.com/paradigmxyz/reth.git", tag = "v0.2.0-beta.6", default-features = false }
-reth-rpc-types-compat = { git = "https://github.com/paradigmxyz/reth.git", tag = "v0.2.0-beta.6", default-features = false }
-reth-revm = { git = "https://github.com/paradigmxyz/reth.git", tag = "v0.2.0-beta.6", default-features = false }
+reth-rpc-types = { git = "https://github.com/paradigmxyz/reth.git", tag = "v0.2.0-beta.7", default-features = false, features = [
+  "arbitrary"
+]  }
+reth-rpc-types-compat = { git = "https://github.com/paradigmxyz/reth.git", tag = "v0.2.0-beta.7", default-features = false }
+reth-revm = { git = "https://github.com/paradigmxyz/reth.git", tag = "v0.2.0-beta.7", default-features = false }
+revm-inspectors = { git = "https://github.com/paradigmxyz/evm-inspectors", rev = "7168ac5" }
```

Comment:
do we need to add this dep? It was previously reexported by `reth-revm`

Reply:
Yes was surprised too but seems they deleted this reexport and they always use the `revm_inspectors` elements, for example here

https://github.com/paradigmxyz/reth/blob/a44e0857373cee447c3823608194ccac53509140/crates/rpc/rpc/src/trace.rs#L33